MovieApp/Screens/More/MoreViewController.swift

```diff
@@ -6,11 +6,172 @@
+		moreTableView.register(UINib(nibName: ConstantsCellMore.myListCell, bundle: Bundle.main), forCellReuseIdentifier: ConstantsCellMore.myListCell)
+		moreTableView.register(UINib(nibName: ConstantsCellMore.standardCell, bundle: Bundle.main), forCellReuseIdentifier: ConstantsCellMore.standardCell)
+	}
+}
+
+// MARK: - IDownloadViewController
+extension MoreViewController: IMoreViewController {
+
+	func showError(_ errorMessage: String) {
+		print(errorMessage)
+	}
+
+	func showMovies() {
+		moreTableView.reloadData()
+	}
+}
+
+// MARK: - UITableViewDataSource
+extension MoreViewController: UITableViewDataSource {
+	func numberOfSections(in tableView: UITableView) -> Int {
+		return viewModel?.sectionsMore.count ?? 0
+	}
+
+	func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		return 1
+	}
+
+	func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		let tabCell = viewModel?.sectionsMore[indexPath.section]
+		switch tabCell {
+		case .profile(let identifer, _):
+			guard let cell = self.moreTableView.dequeueReusableCell(withIdentifier: identifer) as? ProfileTableViewCell else { return UITableViewCell() }
+			cell.configure(with: viewModel?.moreViewModels ?? [])
+			return cell
+
+		case .shared(let identifer, _):
+			guard let cell = self.moreTableView.dequeueReusableCell(withIdentifier: identifer) as? SharedTableViewCell else { return UITableViewCell() }
+			cell.viewModel = self
+			return cell
+
+		case .mylist(let identifer, _):
+			guard let cell = self.moreTableView.dequeueReusableCell(withIdentifier: identifer) as? MyListTableViewCell else { return UITableViewCell() }
+			return cell
+
+		case .appsetting(let identifer, let title):
+			guard let cell = self.moreTableView.dequeueReusableCell(withIdentifier: identifer) as? StandardTableViewCell else { return UITableViewCell() }
+			cell.configure(cellViewModel: title)
+			return cell
+
+		case .account(let identifer, let title):
+			guard let cell = self.moreTableView.dequeueReusableCell(withIdentifier: identifer) as? StandardTableViewCell else { return UITableViewCell() }
+			cell.configure(cellViewModel: title)
+			return cell
+
+		case .help(let identifer, let title):
+			guard let cell = self.moreTableView.dequeueReusableCell(withIdentifier: identifer) as? StandardTableViewCell else { return UITableViewCell() }
+			cell.configure(cellViewModel: title)
+			return cell
+
+		case .signout(let identifer, let title):
+			guard let cell = self.moreTableView.dequeueReusableCell(withIdentifier: identifer) as? StandardTableViewCell else { return UITableViewCell() }
+			cell.configure(cellViewModel: title)
+			return cell
+			
+		case .none:
+			return UITableViewCell()
+		}
+	}
+
+	func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+		switch indexPath.section {
+		case 0:
+			return 200.0
+		case 1:
+			return 200.0
+		default:
+			return 50.0
+		}
```

Comment:
```suggestion
		case 0:
			return 200.0
		case 1:
			return 200.0
		default:
			return 50.0
		}
```
Make an enum or constants describe 0, 1

Reply:
fix complete